activeadmin-chat.gemspec

```diff
@@ -16,10 +16,12 @@ Gem::Specification.new do |s|
 
   s.files = Dir['{app,config,db,lib}/**/*', 'MIT-LICENSE', 'Rakefile', 'README.md']
 
-  s.required_ruby_version = '>= 2.3.0'
```

Comment:
I changed this because motion was raising an exception with ruby 2.4 `NoMethodError: private method `alias_method' called for #<Class:Motion>`
https://travis-ci.org/github/rootstrap/active_admin_chat/jobs/716412253

Reply:
This is a bug in Motion. If it has the code that breaks that Ruby version, it should also have the version requirement.

In this case, I'll probably just rewrite the offending code to keep the same compatibility profile as Rails 5.2.